oauth2_provider/oauth2_backends.py

```diff
@@ -75,6 +75,8 @@ def extract_headers(self, request):
             del headers["wsgi.errors"]
         if "HTTP_AUTHORIZATION" in headers:
             headers["Authorization"] = headers["HTTP_AUTHORIZATION"]
+        if "HTTP_ORIGIN" in headers:
+            headers["Origin"] = headers["HTTP_ORIGIN"]
```

Comment:
Let's document why we're passing origin, we should link to the appropriate OAuthLib docs or code.

Reply:
OAuthLib looks for "origin" in the headers : https://github.com/oauthlib/oauthlib/blob/master/oauthlib/oauth2/rfc6749/grant_types/base.py#L253-L268
We need to translate Django's value